README.md

```diff
@@ -133,8 +133,19 @@ CHANGELOG
 
 Next release:
 
+- [gh-148](https://github.com/flintlib/python-flint/pull/148)
+  Remove debug symbols to make smaller Linux binaries.
+- [gh-144](https://github.com/flintlib/python-flint/pull/144)
+  Add `rel_one_ccuracy_bits` to `arb` and `acb`.
+- [gh-142](https://github.com/flintlib/python-flint/pull/142)
+  Add `acb_theta` (only available for Flint >= 3.1).
```

Comment:
I don't know much about what `acb_theta` is but I get the impression that it warrants a more significant release note (that someone else would need to write).